sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1559,6 +1559,15 @@ object functions {
    */
   def cosh(columnName: String): Column = cosh(Column(columnName))
 
+   /**
+    * @param e hyperbolic angle
+    * @return inverse hyperbolic sine of the given value
+    *
+    * @group math_funcs
+    * @since 3.0.0
+    */
+  def acosh(e: Column): Column = withExpr { Acosh(e.expr) }
```

Comment:
Ur, what about `def acosh(columnName: String)`? Is it omitted intentionally?

Reply:
According to @HyukjinKwon , asinh\acosh\atanh should not be included in this file. missed to remove these. now all removed from functions.scala. thanks for finding this out.

Reply:
Got it. Thank you for the clean ups.